README.md

```diff
@@ -112,16 +128,88 @@ Write a conditional statement that...
 8.  Write your javascript solution below
 ```js
 // your answer here
-```
+
+
+var theNumber = 15 ;
+
+if(theNumber % 3 === 0 ){
+  console.log("Fizz");
+}
+else if(theNumber % 5 === 0){
+  console.log("Buzz");
+}
+else if (theNumber % 3 === 0 && theNumber % 5 === 0)
+{
+  console.log("FizzBuzz");
+}
+else {
+  console.log(theNumber)
+}
 
 #### BONUS
 9.  Research a [loop](https://javascript.info/while-for) so that your condition runs on every number from 0 to 100
-```js
-// your answer here
+var theNumber=1;
+var numberOftimes=100;
+
+for(var i=0;i<numberOftimes;i++) {
+
+if(theNumber % 3 === 0 ){
+  console.log("Fizz");
+  theNumber= theNumber+1;
+}
+else if(theNumber % 5 === 0){
+  console.log("Buzz");
+  theNumber= theNumber+1;
+}
+else if (theNumber % 3 === 0 && theNumber % 5 === 0)
+{
+  console.log("FizzBuzz");
+  theNumber= theNumber+1;
+}
+else {
+  console.log(theNumber)
+  theNumber= theNumber+1;
+}
+
+
+}
+
+
+
+
 ```
 10.  Research a [function](https://javascript.info/function-basics) so that your condition runs on every number from 0 to whatever number is passed into the function
 ```js
-// your answer here
```

Comment:
here is another possible way of writing this code

`const fizzBuzz = function (number){
    for (let i = 0; i <= number; i++){
        if (i % 3 == 0 &&  i % 5 == 0 ) {
            console.log("FizzBuzz")

        } else if (i % 3 == 0) {
            console.log("Fizz")  

        } else if (i % 5 == 0){
            console.log("Buzz");
        }
    }
}`